modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/AbstractDeploymentTestCase.java

```diff
@@ -310,30 +308,8 @@ public static void afterClass() throws IllegalAccessException {
           .dependingOn(exceptionThrowingPlugin);
 
   // Policy file builders
-  protected final PolicyFileBuilder barPolicyFileBuilder =
-      new PolicyFileBuilder(BAR_POLICY_NAME).describedBy(new MulePolicyModelBuilder()
-          .setMinMuleVersion(MIN_MULE_VERSION)
-          .setName(BAR_POLICY_NAME)
-          .setRequiredProduct(MULE)
-          .withBundleDescriptorLoader(
-                                      createBundleDescriptorLoader(BAR_POLICY_NAME,
-                                                                   MULE_POLICY_CLASSIFIER,
-                                                                   PROPERTIES_BUNDLE_DESCRIPTOR_LOADER_ID))
-          .withClassLoaderModelDescriptorLoader(
-                                                new MuleArtifactLoaderDescriptor(MULE_LOADER_ID, emptyMap()))
-          .build());
-  protected final PolicyFileBuilder policyUsingAppPluginFileBuilder =
-      new PolicyFileBuilder(BAR_POLICY_NAME).describedBy(new MulePolicyModelBuilder()
-          .setMinMuleVersion(MIN_MULE_VERSION)
-          .setName(BAR_POLICY_NAME)
-          .setRequiredProduct(MULE)
-          .withBundleDescriptorLoader(
-                                      createBundleDescriptorLoader(BAR_POLICY_NAME,
-                                                                   MULE_POLICY_CLASSIFIER,
-                                                                   PROPERTIES_BUNDLE_DESCRIPTOR_LOADER_ID))
-          .withClassLoaderModelDescriptorLoader(
-                                                new MuleArtifactLoaderDescriptor(MULE_LOADER_ID, emptyMap()))
-          .build());
+  protected final PolicyFileBuilder barPolicyFileBuilder = createPolicyFileBuilder(BAR_POLICY_NAME);
+  protected final PolicyFileBuilder policyUsingAppPluginFileBuilder = createPolicyFileBuilder(APP_PLUGIN_POLICY_NAME);
```

Comment:
The changes in this file are mostly a refactor because I created the [createPolicyFileBuilder](https://github.com/mulesoft/mule/pull/12541/files#diff-10352cf492c370178de8d34ca443355f732a28ca6eed9dcf8b98764887cc5a0dR1203) method to avoid so much repetition.

Additionally, in this particular highlighted fragment you might spot that the `artifactId` of the `policyUsingAppPluginFileBuilder` was fixed, to actually be named according to the resource file that is going to be used for it [here](https://github.com/mulesoft/mule/blob/8a03b5b3f13427d548ba644a96408939539577c8/modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/ApplicationPolicyDeploymentTestCase.java#L540). Otherwise it made no sense to have a different PolicyFileBuilder for it, because it was exactly equal to `barPolicyFileBuilder`.